age-calculator/README.md

```diff
@@ -0,0 +1,31 @@
+### Age Calculator
+
+Age  Calculator App written in HTML, CSS, and JavaScript.
+
+### Use of the Project:
+You can easily calculate your age in years,months and days.
+
+### Used Technologies
+  * HTML5
+  * CSS3
+  * JavaScript
+
+
+#### Steps to Use:
+
+---
+
+- Download or clone the repository
+
+```
+git clone https://github.com/Ayushparikh-code/Web-dev-mini-projects.git
+```
+
+- Go to the directory
+- Run the index.html file
+- Start Calculating!!
+--- 
+
+## Screenshots 
+![age-calci](https://user-images.githubusercontent.com/69195262/124578604-fa787f80-de6b-11eb-9c74-b1dcf3cc0b4f.png)
+
```

Comment:
Use mark-down beautifier.

Use the link -  https://codebeautify.org/markdown-formatter